src/main/java/seedu/main/UI.java

```diff
@@ -24,6 +25,16 @@ public UI() {
         }
     }
 
+    public void printWelcomeMessage() {
+        printMessage("Welcome back! Remember to record your spending today so you can track your spending accurately.");
+        double todaySpending = TransactionList.getTodaySpending();
```

Comment:
Should it be better to take today's spending from the transaction list in our Main class instead of calling a static method? This it quite nested